packages/client/foreman_scap_client_bash/foreman_scap_client_bash.spec

```diff
@@ -22,7 +22,7 @@ Requires: openscap-scanner
 Requires: bzip2
 BuildArch: noarch
 
-Obsoletes: rubygem-foreman_scap_client
+Obsoletes: rubygem-foreman_scap_client < 0.6.3-1
```

Comment:
In Fedora it's custom to only pin to the currently packaged version and increase the release by 1